### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by faddat**
hi :)

So, this is my first pass on the v0.46.x work for Mars.

everything looks good, and all I truly did is fix some whining from the linter :)

### CODE REVIEW

**Review by sontrinh16 — COMMENTED** [review]
seem good so far

File: .golangci.yml [hunk 1]
```diff
@@ -11,9 +11,7 @@ linters:
     - errcheck
     - goconst
     - gocritic
-    - gofmt
     - gofumpt
-    - goimports
```
Comment (larry0x) [inline .golangci.yml:16]: anything wrong with gofmt and goimports that they need to be removed?

File: app/export.go [hunk 2]
```diff
@@ -115,7 +115,7 @@ func (app *MarsApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs [
 		feePool.CommunityPool = feePool.CommunityPool.Add(scraps...)
 		app.DistrKeeper.SetFeePool(ctx, feePool)
 
-		app.DistrKeeper.Hooks().AfterValidatorCreated(ctx, val.GetOperator())
+		app.DistrKeeper.Hooks().AfterValidatorCreated(ctx, val.GetOperator()) //nolint:errcheck // if we explode here, a panic is good
```
Comment (larry0x) [inline app/export.go:118]: if we want to panic on error here, shouldn't be do

```go
if err := app.DistrKeeper.Hooks().AfterValidatorCreated(ctx, val.GetOperator()); err != nil {
	panic(err)
}
```

File: x/gov/keeper/vesting.go [hunk 3]
```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"cosmossdk.io/math"
```
Comment (larry0x) [inline x/gov/keeper/vesting.go:7]: can you name this import `sdkmath`? this is how they name it in the sdk repo. let's be consistent. for example:

https://github.com/cosmos/cosmos-sdk/blob/351916e5be4ca5c35d654434eb4da054d7109291/types/math.go#L4

**Comment by larry0x** [comment]
@faddat btw, i merged #32 into the `sdk-046` branch, which upgrades my two modules to use gov v1 scheme (message-based as opposed to proposal-based)

**Pull request merged**